pydrive2/test/test_fs.py

```diff
@@ -26,16 +27,22 @@ def remote_dir(base_remote_dir):
 
 
 @pytest.fixture
-def fs(tmpdir, base_remote_dir):
-    setup_credentials()
-    auth = GoogleAuth(settings_file_path("default.yaml", tmpdir / ""))
-    auth.ServiceAuth()
+def create_fs(tmpdir, base_remote_dir):
```

Comment:
This `create_fs()` is confusing than `fs` that we had before. Also, `create=False` branch makes it more confusing. 

Can we separate this fixture into `auth` and `fs`, and then wherever you are doing `create_fs(create=False)`, you can do something like follows?

```python
def test(base_remote_dir, auth):
  fs = GDriveFileSystem(base_remote_dir, auth)
```

Reply:
I can rename it a bit, and will see if it makes sense to split it, but we can't do `fs = GDriveFileSystem(base_remote_dir, auth)` in the test, fixture also creates the `base_remote_dir` inside it at the moment, we don't want to repeat this logic inside each test.

Reply:
please review though the actual logic, etc ... so that we are not blocked by these stylistic refactoring

Reply:
@skshetry refactored it, PTAL - was not my intention initially, but it's now way better + I disabled caching in the fs itself (I always forget that it's enabled by default) + added cleanup at the end. Good quick change, thanks.